src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -150,18 +150,26 @@ public override async Task<bool> Execute()
 
                 string manifestDirectoryPath = SaveManifestDirToLocalPath(manifests, this.OutputDir);
 
-                if (!ValidateManifest(manifestDirectoryPath) ||
-                    !Prompt.Confirm(Resources.ConfirmGitHubSubmitManifest_Message) ||
-                    !await this.SetAndCheckGitHubToken())
-                {
+                bool isManifestValid;

```

Comment:
Why not just set the variable here instead of in the if clause?

Reply:
changed to set variable before usage in if clause